### ISSUE TITLE
on async function get error

### ISSUE THREAD

**Issue by avnerbarr** [gsingh93/trace#29]
getting this error on async functions

```
#[trace(logging)]
pub async fn foo(str: &str) {
}

#[trace(logging)]
   |     ----------------- this is not `async`
```

**Comment by gsingh93** [comment]
Thanks for reporting! Just wanted to mention I'm pretty short on time right now so I won't be able to fix this any time soon, but pull requests are welcome.

**Comment by chanq-io** [comment]
Hey @gsingh93 I have a fix for this, can you add me as a collaborator?

**Comment by gsingh93** [comment]
This should be fixed in the latest release (v0.1.7)
